waypoint_following:
  ros__parameters:
    frequency: 30
    odom_topic: /odom
    goal_topic: /goal_pose
    goal_radius: 5.0
    # waypoints: [20.6453, -9.80267, 0., 24.3003, -31.9225, 0., 13.7962, -46.5684, 0., 20.6453, -9.80267, 1., -3.74152, -51.6982, 0., -0.0843143, -0.171923, 0.] # метрические x и y координаты точек
    # waypoints: [20.6453, -9.80267, 0., 24.3003, -31.9225, 0., 13.7962, -46.5684, 0., -3.74152, -51.6982, 0., 20.6453, -9.80267, 1.]
    waypoints: [20.6453, -9.80267, 0., 24.3003, -31.9225, 0., 13.7962, -46.5684, 0., 20.6453, -9.80267, 1.]
    map_frame: map
    start_following: False # Начинать следование только после запуска и инициализации всех нод